apps/condo/domains/news/schema/NewsItemScope.test.js

```diff
@@ -254,5 +259,36 @@ describe('NewsItemScope', () => {
                 },
             )
         })
+
+        test('must throw an error on trying to create/edit scope for published news item', async () => {
+            const [newsItem] = await createTestNewsItem(adminClient, dummyO10n, { isPublished: true })
+
+            await expectToThrowGQLError(
+                async () => await createTestNewsItemScope(adminClient, newsItem),
+                {
```

Comment:
Maybe import errors from schema?

Reply:
I want to test error fields, because they may be received via API by third parties.